server/package.json

```diff
@@ -6,12 +6,13 @@
   "dependencies": {
     "@aws-sdk/client-iot": "^3.78.0",
     "@aws-sdk/client-iot-wireless": "^3.50.0",
-    "brave-alert-lib": "https://github.com/bravetechnologycoop/brave-alert-lib#v7.0.0",
+    "brave-alert-lib": "https://github.com/bravetechnologycoop/brave-alert-lib#td-add-translations",
```

Comment:
Reminder to update this after https://github.com/bravetechnologycoop/brave-alert-lib/pull/46 is merged

Reply:
Done